app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) != 3:
+        return
+    if command.split()[0] != "mv":
+        return
+    cmd, file_from, file_to = command.split()
```

Comment:
You are doing split 3 times. What about creating a new variable for splitted command, and working with that variable in all these lines?